src/Components/DisplayArea/DisplayArea.js

```diff
@@ -59,6 +60,7 @@ const DisplayArea = ({ favorites, recipes, handleFavorites }) => {
   return (
     <section className="display-section">
       {cards}
```

Comment:
When there are cards what is rendered?

Reply:
They appear as a list of individual recipe cards on the page.